### PR TITLE
Cleanup single threaded tests

### PR DESCRIPTION
**Pull request by dain**

### CODE REVIEW

File: concurrent/src/test/java/io/airlift/concurrent/TestBoundedExecutor.java [hunk 1]
```diff
@@ -17,14 +17,14 @@
...
-    @BeforeMethod
+    @BeforeClass
     public void setUp()
             throws Exception
     {
         executorService = Executors.newCachedThreadPool();
     }
 
-    @AfterMethod
+    @AfterClass
```
Comment (electrum) [inline concurrent/src/test/java/io/airlift/concurrent/TestBoundedExecutor.java]: Add alwaysRun to all of these.

**Comment by electrum** [comment]
Looks good

File: concurrent/src/test/java/io/airlift/concurrent/TestBoundedExecutor.java (hunk 1 above)
Comment (dain) [inline concurrent/src/test/java/io/airlift/concurrent/TestBoundedExecutor.java]: do we need that for `@AfterMethod` and `@AfterClass`

**Comment by dain** [comment]
I added `(alwaysRun = true)` to all of the `@AfterMethod` and `@AfterClass`.  Does this look good?

**Comment by electrum** [comment]
I think you need it on anything that does cleanup if you want the cleanup to happen even if the test fails. Looks good.

**Pull request merged**